packages/discovery-react-components/src/components/TooltipHighlight/TooltipHighlight.tsx

```diff
@@ -96,6 +102,12 @@ export function calcToolTipContent(
   if (facetInfoMap[facetId]) {
     enrichColor = facetInfoMap[facetId].color;
     enrichFacetDisplayname = facetInfoMap[facetId].displayName;
+    if (
+      enrichFacetDisplayname.localeCompare(enrichValue, undefined, { sensitivity: 'base' }) == 0
+    ) {
+      // This case applies to keyowrds
+      enrichFacetDisplayname = 'Keyword';
```

Comment:
First, this should be "Keywords" (plural), as that's what the enrichment is called. Second, at the very least, this should be a constant (at the top of the file or within its own file with other constants) instead of a bare string in the middle of the code.

Reply:
Updated